package.json

```diff
@@ -27,6 +27,7 @@
     "test:debug": "mocha --inspect-brk"
   },
   "dependencies": {
+    "ansi-html": "^0.0.7",
```

Comment:
Eeck, 0.0.7 seems rough (though this package has ~ 4 million weekly downloads)

Reply:
ya, likely fine :P

Reply:
I borrowed it from webpack's [dev server](https://github.com/webpack/webpack-dev-server/blob/90d4a7c7ae4198b79ff03a67b0037e4bed3154e8/client-src/default/overlay.js#L6). There's a comment there that tells

> // The error overlay is inspired (and mostly copied) from Create React App (https://github.com/facebookincubator/create-react-app)
// They, in turn, got inspired by webpack-hot-middleware (https://github.com/glenjamin/webpack-hot-middleware).

😆